tools/orca/Cargo.toml

```diff
@@ -19,9 +19,10 @@ doctest = false
 
 [dependencies]
 async-trait = { workspace = true }
+chrono.workspace = true
```

Comment:
`<crankyoldman>`
please use the same TOML format as the other lines
`</crankyoldman>` 😄 Thanks!

Reply:
😂, sure!